blueprints/sl-ember-components/index.js

```diff
@@ -2,29 +2,17 @@ module.exports = {
     afterInstall: function() {
         const self = this;
 
-        return this.addBowerPackageToProject( 'bootstrap-datepicker', '~1.3.0' )
-            .then( function() {
-                return self.addBowerPackageToProject( 'fontawesome', '~4.3.0' );
-            })
-            .then( function() {
-                return self.addBowerPackageToProject( 'highcharts', '~4.1.5' );
-            })
-            .then( function() {
-                return self.addBowerPackageToProject( 'moment', '~2.10.3' );
-            })
-            .then( function() {
-                return self.addBowerPackageToProject( 'moment-timezone', '~0.4.0' );
-            })
-            .then( function() {
-                return self.addBowerPackageToProject( 'select2', '~3.5.2' );
-            })
-            .then( function() {
-                return self.addBowerPackageToProject( 'typeahead.js', '~0.11.1' );
-            })
-            .then( function() {
-                return self.addBowerPackageToProject( 'softlayer/sl-bootstrap', '1.1.0' );
-            });
+        return this.addBowerPackagesToProject([
```

Comment:
`ember install sl-ember-components` is the incorrect command to run, as it will look at the NPM registry and install version v0.8.0 of this addon and not your branch.  Should instead use `ember install erangeles/sl-ember-components#issue342`